pages/_app.js

```diff
@@ -31,6 +34,11 @@ const FontFix = createGlobalStyle`
   }
 `
 
+i18next.init({
+  lng: 'de',
+  resources: { de: { translation: de } }
+})
+
```

Comment:
You probably want to use a more intelligent backend for that since this would always include the translation files for all languages. See e.g. the implementation in serlo.org